common/src/main/java/com/viaversion/viaversion/protocols/v1_8to1_9/storage/EntityTracker1_9.java

```diff
@@ -185,7 +191,10 @@ public void handleEntityData(int entityId, List<EntityData> entityDataList) {
                     byte data = (byte) entityData.getValue();
                     if (entityId != getProvidedEntityId() && Via.getConfig().isShieldBlocking()) {
                         if ((data & 0x10) == 0x10) {
-                            if (validBlocking.contains(entityId)) {
+                            // If sword blocking is done through consumables, don't add a shield. We can use
+                            // 1.20.5 here because consumables display properly in 3rd person even in 1.20.5-1.21.3
+                            if (validBlocking.contains(entityId) &&
+                                    user().getProtocolInfo().protocolVersion().olderThan(ProtocolVersion.v1_20_5)) {
```

Comment:
Why are you checking for 1.20.5 here. You removed the 1.20.5 consumable blocking fix, so you would need the old fix for 1.20.5 as well

Reply:
You're right, I think this was leftovers from some experiments I did earlier. Indeed, without the change 1.20.5-1.21.3 clients can't see shields on other players. 

Thanks!